tests/test_other.py

```diff
@@ -9970,3 +9970,56 @@ def test_relocatable_limited_exports(self):
     self.assertGreater(len(exports_linkable), 1000)
     self.assertIn('sendmsg', exports_linkable)
     self.assertNotIn('sendmsg', exports)
+
+  @is_slow_test
+  def test_deps_info(self):
+    # Verify that for each symbol listed in deps_info all the reverse
+    # dependencies are indeed valid.
+    # To do this we compile a tiny test program that depend on the address
+    # of each function.  Once compiled the resulting JavaScript code should
+    # contain a reference to each of the dependencies.
+    start_at = None
+    assert not start_at or start_at in deps_info.deps_info
```

Comment:
this assertion at this location will never fail, as the var is assigned to None right before it.

Reply:
Added a comment about what this does.